test/Microsoft.ML.TestFramework/Microsoft.ML.TestFramework.csproj

```diff
@@ -22,6 +23,6 @@
     <NativeAssemblyReference Include="MklProxyNative" />
     <NativeAssemblyReference Include="MatrixFactorizationNative" />
     <NativeAssemblyReference Include="MklImports" />
-    <NativeAssemblyReference Condition="'$(OS)' == 'Windows_NT'" Include="libiomp5md"/>
+    <NativeAssemblyReference Condition="'$(OS)' == 'Windows_NT'" Include="libiomp5md" />
   </ItemGroup>
```

Comment:
Minor nitpick, but this line is unchanged except for the space which probably was added mistakenly. Can you please delete the space and leave the line unchanged?